sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystTypeConverters.scala

```diff
@@ -421,6 +421,17 @@ object CatalystTypeConverters {
     case other => other
   }
 
+  def convertToCatalystUnsafe(a: Any): Any = a match {
```

Comment:
Actually do we want to have this in `CatalystTypeConverters`?

I think it should be in `ExpressionEvalHelper.scala`.